resources/assets/components/Tags/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { map } from 'lodash';
+
+class Tags extends React.Component {
+  render() {
+
+    return (
+        <ul className="form-actions -inline">
+          <li><b> Tags </b></li>
+          <br/>
+          <li> <input className="button -secondary" value="Good Photo"/></li>
+          <li> <input className="button -secondary" value="Good Quote"/></li>
+          <br/>
```

Comment:
`ul` elements should only have `li` inside them, so spacing like this should be handled with CSS if necessary.